web/src/containers/Nav.js

```diff
@@ -1,24 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { VerticalNav } from '@cmsgov/design-system-core';
 import { connect } from 'react-redux';
 import NavLink from '../components/NavLink';
+import { generateKey as actualGenerateKey } from '../util'
 
-const Nav = ({ links, selectedId }) => (
-  <VerticalNav
-    component={NavLink}
-    items={links}
-    selectedId={selectedId} />
-);
+const Nav = ({ generateKey, items, location }) => {
+  // force component update when location changes
+  const [key, setKey] = useState('');
+  useEffect(() => setKey(generateKey()), [location]);
+  return (
+    <VerticalNav
+      component={NavLink}
+      items={items}
+      key={key}
```

Comment:
The `<VerticalNav/>` component doesn't update when changing the `defaultCollapsed` property within `items`. A workaround is to force an update with a `generatedKey` when the location changes.